public_html/includes/js/develop/createForm_form.js

```diff
@@ -1531,8 +1531,9 @@ function addMetadataStandard(val){
 	}
 
 	var options    = metadataSchema;
-	var identifier = val.split(' : ')[1];
-	var standard   = val.split(' : ')[0];
+	val = val.replace(/\s+/g, '');
```

Comment:
I don't understand the question.

This is the actual data for the form. It isn't interacted with any other place inside MFCS right now. (Though, there is no reason it couldn't be if it is needed). This information is required for exporting to certain systems ... such as for OAI (it was added for doing OAI and other imports into WMS).